### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by tobil4sk**
Clean up integration tests' output, and add test configuration for vscode.

### CODE REVIEW

File: test/TestBase.hx [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+	static final haxelibPath = FileSystem.fullPath("run.n");
+
 	public function runHaxelib(args:Array<String>, printProgress = false) {
-		var p = new Process("neko", [haxelibPath].concat(args));
+		final p = new Process("neko", [haxelibPath].concat(args));
 		var stdout = '';
 		var stderr = '';
 		var eofCount = 0;
-		var c;
+		var c:Int;
```
Comment (Simn) [inline test/TestBase.hx:15]: Was this necessary for some reason?

File: test/tests/TestSemVer.hx [hunk 2]
```diff
@@ -61,12 +62,9 @@ class TestSemVer extends TestBase {
...
 
-	function parseInvalid( str:String ) {
-		return try {
-			(SemVer.ofString( str ) : String);
-		} catch (e:String) {
-			"invalid";
-		}
+	function parseInvalid( str:String ):String {
+		return try SemVer.ofString(str)
+			catch (e:String) "invalid";
```
Comment (Simn) [inline test/tests/TestSemVer.hx:67]: Indentation looks weird here.

File: test/HaxelibTests.hx [hunk 3]
```diff
@@ -20,38 +19,37 @@ class HaxelibTests {
...
 		if (!FileSystem.exists(dir)) return;
-		var exitCode = switch (Sys.systemName()) {
+		final exitCode = switch (Sys.systemName()) {
 			case "Windows":
 				Sys.command("rmdir", ["/S", "/Q", StringTools.replace(FileSystem.fullPath(dir), "/", "\\")]);
 			case _:
 				Sys.command("rm", ["-rf", dir]);
 		}
-		if (exitCode != 0) {
+		if (exitCode != 0)
```
Comment (Simn) [inline test/HaxelibTests.hx:40]: Personally, I don't like this brace-less style. But if this what the rest of haxelib does then I agree that we should make it consistent.

File: test/tests/TestSemVer.hx (hunk 2 above)
Comment (tobil4sk) [reply test/tests/TestSemVer.hx]: Reverted

File: test/HaxelibTests.hx (hunk 3 above)
Comment (tobil4sk) [reply test/HaxelibTests.hx]: I had a look through and it does seem like the braces are preferred. Might be a good idea to setup a haxe checkstyle file for stuff like this at some point.

File: test/TestBase.hx (hunk 1 above)
Comment (tobil4sk) [reply test/TestBase.hx]: No, reverted it now.

**Pull request merged**